sycl/doc/UsersManual.md

```diff
@@ -350,6 +350,18 @@ and not recommended to use in production environment.
 
 ## Other options
 
+**`-f[no-]offload-fp32-prec-sqrt`**
```

Comment:
@gmlueck during writing this document I've realized, that there is a [similar option](https://github.com/intel/llvm/pull/17033/files#diff-cbe0f44959df59a646e8857e1a118f8480a4a965b4e47d3e965e4904b0f2092cR408) already, that works only for CUDA and HIP. I believe, that these options should be merged under sycl-fp32-prec-sqrt name and div option should be renamed to sycl-fp32-prec-div (unless we want to deprecate sycl naming for them, which also might make sense in a certain cases). WDYT?

Reply:
I have no strong feelings on which option should remain (offload or sycl), but I do agree that we should merge these.